src/store/initial-state.test.js

```diff
@@ -157,4 +156,15 @@ describe('getInitialState', () => {
     });
     window.localStorage.clear();
   });
+
+  it('hides parameters when parameter flag is true', () => {
+    const state = getInitialState({
+      ...props,
+      flags: {
+        parameters: true,
+      },
+    });
+    const parametersDisabled = state.nodeType.disabled.parameters;
+    expect(parametersDisabled).toBe(true);
```

Comment:
Thanks for adding this test in! 

One suggestion: this test is based on the assumption that assigning `nodeType.disabled.parameters` will automatically ensure parameters are not displayed within the app. Once we have removed the 'hack' in mockstate and go further into fixing the tests, we definitely need to update the tests under `actions/nodes.test.js` to ensure parameters nodes are not returned within the actions and flowchart.